php_phongo.c

```diff
@@ -1880,11 +1878,17 @@ PHP_MINIT_FUNCTION(mongodb)
 {
 	(void)type; /* We don't care if we are loaded via dl() or extension= */
 
+	char *php_version_string = malloc(4 + sizeof(MONGODB_VERSION_S) + 1);
```

Comment:
```mongoc_handshake_data_append()``` frees it. This is why the handshake is added *before* the libbson memory vtable is set-up too.